lib/pymedphys/cli/dev.py

```diff
@@ -14,8 +14,10 @@ def dev_cli(subparsers):
 def add_docs_parser(dev_subparsers):
     parser = dev_subparsers.add_parser("docs")
 
-    parser.add_argument("--live", help="Make the docs live reload", action="store_true")
```

Comment:
presumably the Jupyter Book is inherently "live"?

Reply:
Jupyter book is currently missing the live functionality. But with caching built in of the notebooks rebuilding is very quick.